### PR TITLE
Update 404 error page

### PR DESCRIPTION
**Pull request by msmanoj**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1325

## Description
- Updated the 404 error page to match with the new design

## Subsequent tasks
- Log the 404s in the service monitoring system being developed by BE (https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1360)

## Deployment URL
http://update-404-page.review.ensembl.org/hello

### CODE REVIEW

File: src/ensembl/src/shared/components/error-screen/ErrorScreen.scss [hunk 1]
```diff
@@ -46,7 +46,7 @@
   font-size: 12px;
```
Comment (azangru) [inline src/ensembl/src/shared/components/error-screen/ErrorScreen.scss:46]: You will find above that the `.generalErrorScreen` class has a height of `100vh`. This is too much for the 404 page when it is rendered client-side, with the top bar and the launchbar. Currently, when the 404 page is server-rendered, it does not have the topbar and the launchbar on the top; so the 100vh works. I suppose we will want to include the topbar and the launchbar into the server-rendered payload, but we don't yet have clarity on that.

Anyhow, just wanted to let you know. Not sure what the proper solution should be; there are several options.

Comment (msmanoj) [reply src/ensembl/src/shared/components/error-screen/ErrorScreen.scss:46]: Changed `100vh` to `100%` as discussed during code review

**Comment by jyothishnt** [comment]
All good. One doubt - When I go to http://update-404-page.review.ensembl.org/help and then change the url to http://update-404-page.review.ensembl.org/helpx  I see the help icon still selected in the launchbar. Is this how it is?

**Review by jyothishnt — APPROVED** [review]

**Comment by azangru** [comment]
> and then change the url

The problem is a bit more general: if you enter an invalid url starting with the same string as any of our existing apps (`helpx`, `genome-browser-foo`, or whatever), a button in the launchbar will light up as selected. This is because we are using a regex in `LaunchbarButton` to check whether an app is matched ([link](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/header/launchbar/LaunchbarButton.tsx#L39-L40)). We could instead look inside redux, where we store the current app name. This can be fixed in this or in a separate PR.

**Review by azangru — APPROVED** [review]

**Pull request merged**